impl/client/src/main/javascript/web/pentaho/visual/ccc/barAbstract/model.js

```diff
@@ -18,24 +18,25 @@ define([
   "../categoricalContinuousAbstract/model",
   "pentaho/i18n!../abstract/i18n/model",
   "../abstract/mixins/settingsMultiChartType"
-], function(module, categoricalContinuousAbstractModelFactory, bundle, settingsMultiChartType) {
+], function(module, baseModelFactory, bundle, settingsMultiChartType) {
```

Comment:
That's a good point. How do you propose things could be structured?
`pentaho/visual/stock/bar` and `pentaho/visual/stock/bar/ccc/view` ?